mappings/net/minecraft/block/FrostedIceBlock.mapping

```diff
@@ -1,5 +1,7 @@
 CLASS net/minecraft/unmapped/C_gxtlheps net/minecraft/block/FrostedIceBlock
 	FIELD f_dvmsaosj AGE Lnet/minecraft/unmapped/C_vltzvhxi;
+	FIELD f_lvfudhvm NEIGHBORS_TO_AGE I
```

Comment:
What is this

Reply:
again similar to the dripstone one, i can't find usages so I'm not overly sure